app/src/main/java/com/kickstarter/ui/activities/HelpSettingsActivity.kt

```diff
@@ -46,21 +46,10 @@ class HelpSettingsActivity : BaseActivity<HelpSettingsViewModel.ViewModel>() {
                     .subscribe(this::composeContactEmail)
         }
 
-        cookie_policy.setOnClickListener{
-            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.CookiePolicy::class.java))
-        }
-
-        help_center.setOnClickListener {
-            startActivityWithSlideLeftTransition(Intent(Intent.ACTION_VIEW, Uri.parse(Secrets.HelpCenter.ENDPOINT)))
-        }
-
-        privacy_policy.setOnClickListener {
-            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.Privacy::class.java))
-        }
-
-        terms_of_use.setOnClickListener {
-            startActivityWithSlideLeftTransition(Intent(this, HelpActivity.Terms::class.java))
-        }
+        cookie_policy.setOnClickListener{ loadChromePages("cookies") }
```

Comment:
These should be constants. We previously used `Secrets.HelpCenter.ENDPOINT` and the static classes in `HelpActivity`